eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -34,6 +38,24 @@ foreach ($res in $responses)
   }
   $branch = $res.ref
   $branchName = $branch.Replace("refs/heads/","")
+  $branchName -match $CentralPRRegex | Out-Null
```

Comment:
This entire block should be conditionally ran. It isn't a requirement that we have a CentralRepoId and Regex that is just an option. If those aren't specified then we should continue with our existing workflow.